docker-compose.yml

```diff
@@ -10,3 +10,9 @@ services:
       MYSQL_USER:          'bacterial_growth'
       MYSQL_PASSWORD:      'nVI8imBD3bl24pcP'
       MYSQL_ROOT_PASSWORD: 'nVI8imBD3bl24pcP'
+  redis:
+    image: redis:7.4.2
+    restart: always
+    container_name: bacterial_growth_redis
+    ports:
+      - "6380:6379"
```

Comment:
The default redis port is 6379, but I'm using that port +1 to avoid potential conflicts with other redis instances on the server, the same thing I'm doing with the mysql port to avoid conflicting with microbetag.